autofs-5.0.4 - use misc device

From: Ian Kent <raven@themaw.net>

Change default installed configuration to use the misc device ioctl
interface if available.
---

 CHANGELOG                      |    1 +
 redhat/autofs.sysconfig.in     |    2 +-
 samples/autofs.conf.default.in |    2 +-
 3 files changed, 3 insertions(+), 2 deletions(-)


diff --git a/CHANGELOG b/CHANGELOG
index da7ecbf..96a4617 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -32,6 +32,7 @@
 - another easy alloca replacements fix.
 - add LSB init script parameter block.
 - fix file map lookup when reading included or nsswitch sources.
+- use misc device ioctl interface by default, if available.
 
 4/11/2008 autofs-5.0.4
 -----------------------
diff --git a/redhat/autofs.sysconfig.in b/redhat/autofs.sysconfig.in
index 04e521c..97e20fe 100644
--- a/redhat/autofs.sysconfig.in
+++ b/redhat/autofs.sysconfig.in
@@ -110,7 +110,7 @@ BROWSE_MODE="no"
 # If the kernel supports using the autofs miscellanous device
 # and you wish to use it you must set this configuration option
 # to "yes" otherwise it will not be used.
-#USE_MISC_DEVICE="no"
+USE_MISC_DEVICE="yes"
 #
 #OPTIONS=""
 #
diff --git a/samples/autofs.conf.default.in b/samples/autofs.conf.default.in
index 52d18ec..62084c2 100644
--- a/samples/autofs.conf.default.in
+++ b/samples/autofs.conf.default.in
@@ -110,7 +110,7 @@ BROWSE_MODE="no"
 # If the kernel supports using the autofs miscellanous device
 # and you wish to use it you must set this configuration option
 # to "yes" otherwise it will not be used.
-#USE_MISC_DEVICE="no"
+USE_MISC_DEVICE="yes"
 #
 #OPTIONS=""
 #
